src/main/java/course_project/sea_battle/Main.java

```diff
@@ -0,0 +1,12 @@
+package course_project.sea_battle;
+
+import course_project.sea_battle.conroller.SeaBattle;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
```

Comment:
Strange... 
![image](https://user-images.githubusercontent.com/58872509/134204400-849e6ad7-342b-487c-b6c9-89d1018c3360.png)

Reply:
Well-done! The game is quite comfortable to play, it has a nice interface. Also, you show good knowledge of Java Core and Java libraries, mostly abstractions are chosen correctly, and the structure of the project is quite cool. Here and there are some minor Code Conventions violations, but in general the code is easy to read. Cool implementations of Speaker and Threads! 

Approved